MoreCyclopsUpgrades/CyclopsUpgrades/BatteryCyclopsUpgradeHandler.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace MoreCyclopsUpgrades.CyclopsUpgrades
+{
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    internal class BatteryCyclopsUpgradeHandler : ChargingUpgradeHandler
+    {
+        public delegate void BatteryDrainedEvent(BatteryDetails details);
+        public BatteryDrainedEvent OnBatteryDrained;
+
+        internal const float Mk2ChargeRateModifier = 1.15f; // The MK2 charging modules get a 15% bonus to their charge rate.
```

Comment:
Round up all the copied `const` values and point them back to the `PowerManager`